libraries/monitor/src/logger.js

```diff
@@ -0,0 +1,119 @@
+const os = require('os');
+const assert = require('assert');
+const stringify = require('fast-json-stable-stringify');
+
+const LEVELS = {
+  emerg: 0,
+  alert: 1,
+  crit: 2,
+  err: 3,
+  warning: 4,
+  notice: 5,
+  info: 6,
+  debug: 7,
+};
+
+/*
+ * Implements the mozlog standard as defined in
+ * https://wiki.mozilla.org/Firefox/Services/Logging
+ *
+ * We can consider supporting extra logging standards
+ * later if we want.
+ */
+class Logger {
```

Comment:
Any chance we could have child loggers?  I suspect being able to pass in a set of fields to the constructor and returning a new Logger instance from a child method could achieve this?

Reply:
Child loggers are supported at the lib-monitor level via `prefix(<name>, {extra-stuff})`. Each child ends up with its own `Logger`. I just did it that way since that is the way we supported children before so it will be easier to port.

Reply:
cool!